about.html

```diff
@@ -3,21 +3,93 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title> About Document</title>
+    <link rel="stylesheet" href="css/about.css">
+
 </head>
 <body>
+    <div id="nav-container">
+    <header>
+        <nav aria-label="primary-navigation"></nav>
+    </div>
     
-<h1>Misson Statement</h1>
- <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Non inventore dignissimos officiis adipisci eveniet neque totam iusto facilis, hic quae ex eligendi ipsa est, maxime cupiditate pariatur soluta error libero.</p>
+    </header>
 
+    </div>
+    <div id="mission-container">
+        <div id="mission-heading">
+        <h1><b>Mission Statement</b></h1>
+        </div>
+        <p>The mission of the MESA Program at College of the Canyons is to assist educationally underrepresented studentsin understanding and navigating the STEM transfer process in order to successfully graduate with a STEM degree, transfer to a four-year institution, and ultimately becomesignificant and positive contributors to the socioeconomic prosperity of their communities.</p>
+    </div>
 
- <img src="" alt="">
- <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Placeat dolorem dicta laudantium laboriosam sapiente ex, temporibus impedit! Quo officia molestiae, hic itaque labore quibusdam facere consequatur nobis, voluptatum quod culpa.</p>
+    
+    <div class="biography-container">
```

Comment:
There are two ways we can make this layout better and more responsive:

1. As the viewport expands, try to add more whitespace on the left and right of the containers.
2. Put the biography containers into a grid using `display: grid`. This may be difficult, I suggest looking at resources like w3schools or MDN Web Docs to learn how to use grid.